.jenkins/infrastructure/build_azure_managed_images.Jenkinsfile

```diff
@@ -7,7 +7,7 @@ import java.time.format.DateTimeFormatter
 OECI_LIB_VERSION = env.OECI_LIB_VERSION ?: "master"
 oe = library("OpenEnclaveCommon@${OECI_LIB_VERSION}").jenkins.common.Openenclave.new()
 
-GLOBAL_TIMEOUT_MINUTES = 240
+GLOBAL_TIMEOUT_MINUTES = 480
```

Comment:
I don't see this change mentioned in the PR description.   Is this intentional?  Is it intended to be permanent?  Or a workaround for some other issue #?

Reply:
I was asked by @BRMcLaren to carry this changes as we need to build a new image (based on this PR) that includes the newly introduced dependencies.

It would be good for Brett to comment on this.